lib/locasms/version.rb

```diff
@@ -1,3 +1,3 @@
 module LocaSMS
-  VERSION = "0.1.5"
+  VERSION = '0.2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.